### PR TITLE
[Feature] Support support and class-wise evaluation results

### PR DESCRIPTION
**Pull request by LXXXXR**
Hi,
This pull request supports support in evaluation metrics and modifies other metrics to support class-wise outputs.
FYI: https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_recall_fscore_support.html#sklearn.metrics.precision_recall_fscore_support
Previously, the file eval_metrics.py was misplaced, therefore was moved in this pr.

Please kindly have a look. Thanks.

### CODE REVIEW

File: mmcls/core/evaluation/__init__.py [hunk 1]
```diff
@@ -1,8 +1,9 @@
 from .eval_hooks import DistEvalHook, EvalHook
+from .eval_metrics import f1_score, precision, recall, support
```
Comment (MeowZheng) [inline mmcls/core/evaluation/__init__.py:2]: import calculate_confusion_matrix

File: mmcls/core/evaluation/eval_metrics.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
+import numpy as np
+import torch
+
+
+def calculate_confusion_matrix(pred, target):
```
Comment (MeowZheng) [inline mmcls/core/evaluation/eval_metrics.py:5]: add docstring

**Review by MeowZheng — APPROVED** [review]

**Review by ycxioooong — COMMENTED** [review]
Could you please check the cases not covered by unit tests?

**Comment by LXXXXR** [comment]
> Could you please check the cases not covered by unit tests?
> 
> We may need to add tests to cover these lines.

Done.

**Pull request merged**